Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN mkdir -p /srv/node
 ADD package.json /srv/node/
 WORKDIR /srv/node
 
+COPY docker/yarnpkg.gpg.key /tmp/yarnpkg.gpg.key
```

Comment:
Could you add a comment saying that this lives at https://dl.yarnpkg.com/debian/pubkey.gpg ? It would be bizarre if we ever had to update it so I don't think it's a problem that we are making a local copy.